bin/copy-public.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -ex
+
+cp -r /opt/kimai/public /
```

Comment:
When using 

```yml
autoupdate:
    image: kimai/kimai2:fpm-alpine-1.11.1-prod
    volumes:
        - web:/opt/kimai/public
        - ./copy-public.sh:/copy-public.sh
    entrypoint: /copy-public.sh
```

It gives the following permission error:

```sh
+ cp -r /opt/kimai/public /
cp: can't create directory '/public': Permission denied
```

Reply:
I'll double check, it worked for me.

Reply:
It seems the user `www-data` used in the kimai Docker image has no permissions and therefore the image for the autoupdate needs to be run as root user (`user: 0:0`):

```yml
autoupdate:
    image: kimai/kimai2:fpm-alpine-1.11.1-prod
    volumes:
        - web:/opt/kimai/public
        - ./copy-public.sh:/copy-public.sh
    entrypoint: /copy-public.sh
    user: 0:0
```